### ISSUE TITLE
IE arrow functions

### ISSUE THREAD

**Issue by HamAndRock** [paleo/direct-vuex#44]
Hello,
I have been using this amazing package for awhile. But I am struggling with IE 11 support. After compilation arrow functions => are left in code.

I managed to fix this by using ems version of files instead of umd. If i swap "module" and "main" in your package.json the compilation works perfectly and it runs on IE without any problems.

![image](https://user-images.githubusercontent.com/24235083/78842401-26564c80-7a00-11ea-8128-4b7ddcd09c9c.png)

I don't really understand how things like EMS and UMD work, but swapping those values fixed my issue, but I don't think its the correct way to fix it :D

Thanks for any suggestions and for your work on this amazing package, HamAndRock.

**Comment by paleo** [comment]
You are right. It is the way I've chosen:

1. A legacy compilation to ES5, packed in a bundle in the legacy UMD format;
2. A more modern compilation to ES6, packed in a bundle as a modern ES6 module ("esm").

I could produce legacy JS code for IE in an ES6 module but it seems less logical. If you need to build your application for IE, the UMD bundle is the way to go. When this situation will cause a problem in your build process, feel free to reopen this ticker or create a new one. :-)

**Comment by HamAndRock** [comment]
Thanks for answer, how do specify which .js file should I use with compilation ? Because if I import the .ts types I get TS support if I import the .js files I don't have the WebStorm whispering feature. I managed to fix it by replacing the 'library string.ts' on build with the .js file but that is not the correct way to fix this issues I guess.

I know this isn't your problem but could you help me ? 

Thanks a lot HamAndRock.

**Comment by robinchow** [comment]
@HamAndRock not sure if this is applicable to you, but I'll detail my setup in case it is.

We use Webpack to compile and bundle our application. We've left out build target as the default, which is _web_ (ref: https://webpack.js.org/configuration/target/). That means by default webpack will find a module, looks inside its _package.json_ and picks the first out of _browser_, _module_ and _main_ (ref: https://webpack.js.org/configuration/resolve/#resolvemainfields)

For _direct-vuex_ that means it will take the file point to by _module_ and compile with *dist/direct-vuex.esm.js*, since the package.json states:

> "main": "dist/direct-vuex.umd.js",
> module": "dist/direct-vuex.esm.js",

We didn't want to change direct-vuex's package.json, since we have automated builds etc, so instead, we updated our own webpack config to force it to use the umd file.

```javascript
modules.export = {
  resolve: {
    alias: {
      'direct-vuex$': 'direct-vuex/dist/direct-vuex.umd.js'
    }
  }
}
```
